README.md

```diff
@@ -19,9 +19,86 @@ Assisted close reading tool
+
+- [Docker desktop](https://docs.docker.com/docker-for-windows/install/)
+
+### Step 1
+
+First test that the docker installation is working. Open a Powershell prompt (press Windows+S and type Powershell) and run:
+
+```shell
+docker run hello-world
+```
+
+This should show a message that your Docker installation is working correctly. If so, we can proceed to the installation of evidence tool, otherwise we suggest to check the [Docker troubleshooting page](https://docs.docker.com/docker-for-windows/troubleshoot/).
+
+### Step 2
+
+[Download](https://github.com/ADAH-EviDENce/evidence/archive/master.zip) a copy of evidence archive and extract its contents on your machine.
+
+### Step 3
+
+Open a Powershell prompt:
+Change your current working directory to where you extracted the files. For instance:
+
+```shell
+cd C:\Users\JohnDoe\Downloads\evidence-master\evidence-master
+```
+
+The demo can be started with the below command. Keep this Powershell window open and running during the demo.
+
+```shell
+$Env:EXPERIMENT="demo"
+docker-compose up --build
+```
+
+The command above downloads necessary Docker images, builds all the Docker images and starts the demo.
+
+The command prints many log messages. If all goes well, the last lines of the output should be:
+
+```shell
+...
+indexer_1        | Indexing done.
+evidence-master_indexer_1 exited with code 0
+```
+
+In some cases, the docker containers cannot access the necessary folder. In this case, you get output like the following:
+
+```powershell
+PS C:\Users\JohnDoe\evidence-master> docker-compose up --build
+Creating network "evidence-master_default" with the default driver
+Building server
+Step 1/28 : FROM golang:1.12-stretch as buildserver
+ ---> 563601c9e3b2
+...
+Creating evidence-master_elasticsearch_1 ... done                                                                               Creating evidence-master_indexer_1       ... error
+ERROR: for evidence-master_indexer_1  Cannot create container for service indexer: status code not OK but 500: {"Message":"Unhandled exception: Filesharing has been cancelled",...
+
+ERROR: for evidence-master_server_1  Cannot create container for service server: status code not OK but 500: {"Message":"Unhandled exception: Filesharing has been cancelled",...
+
+ERROR: for indexer  Cannot create container for service indexer: status code not OK but 500: {"Message":"Unhandled exception: Filesharing has been cancelled",...
+
+ERROR: for server  Cannot create container for service server: status code not OK but 500: {"Message":"Unhandled exception: Filesharing has been cancelled",...
+ERROR: Encountered errors while bringing up the project.
+```
+
+This can be solved by explicitly giving docker access to it. Do this by opening the Docker dashboard by right-clicking the docker icon in the Windows taskbar. Then, go to Settings/Resources/FILE SHARING and add the folder where you extracted the files before, and try running the command above again.
```

Comment:
```suggestion
This can be solved by explicitly giving Docker access to the folder. You can do this by:
1. opening the Docker dashboard by right-clicking the Docker icon in the Windows taskbar 
2. go to Settings/Resources/FILE SHARING and add the folder where you extracted the files before
3. try running the command again:
```shell
$Env:EXPERIMENT="demo"
docker-compose up --build
```
```